src/organization/organization.service.ts

```diff
@@ -299,4 +300,58 @@ export class OrganizationService {
     };
     return json?.Data?.[0]?.['subOrgs'] || [];
   }
+
+  private async getOrgDefinitionByUid(uid: string) {
```

Comment:
Minor thing, but would be nice to have private methods after public in the file

Reply:
👍